fab/fabfile.py

```diff
@@ -728,6 +730,18 @@ def force_update_static():
     execute(services_restart)
 
 
+def _tag_commit():
+    sh.git.fetch("origin", env.code_branch)
```

Comment:
I don't know of a better way. We do this in other places as well. We could implement a function that gets (or asserts) the remote name by repo URL, but that probably has edge cases too.